### ISSUE TITLE
Upgrade to Polymer 1.0

### ISSUE THREAD

**Issue by rosepriya-jk** [ionelmc/polymer-select-box#5]
Is the element migration to Polymer 1.0 in-progress?

**Comment by ionelmc** [comment]
Yes, I plan to do that. Just need to find some free time to upgrade all my polymer stuff.

**Comment by M0nter0** [comment]
Thanks ionelmc for your effort. Do you have any idea about when it could be ready?

**Comment by ionelmc** [comment]
Soon (tm) :grimacing: 

I hope to get some time in August for this ...
